### ISSUE TITLE
Sprayer vehicles wont proceed when driving beside another vehicle

### ISSUE THREAD

**Issue by resultwrap** [Courseplay/courseplay#6685]
Hello all, I've been having this issue for a few months now. I just never said anything about it because I thought the issue would've been caught already and maybe fixed but I haven't seen any fixes on the issue so I figured maybe no one else caught the issue.

When I have a sprayer vehicle spraying in a field, I have a course I made going back to refill when it's empty. Every time the sprayer vehicle goes back to refill, it's as if the sprayer boom is still open and wont proceed after it encounters a parked vehicle which is clearly out of the way. Other random objects and implements don't cause this issue, only parked vehicles until I get in the vehicle and move it a little then it starts to move. I did a test with all the sprayer vehicles I have and it seems to only happen on sprayer vehicles. Not pull behind sprayers. See video attached. Also, I just noticed this as I was recording the video, the lights on the vehicles turn on and off randomly, that is not me doing that. So I'm not really sure what's going on there either...

I am always updated to the latest versions of courseplay.

I apologize for no log because I'm not really sure what channels to engage for this issue, and I'm not entirely sure how they work? would I have to engage on each vehicle or just one vehicle...

P.S. Thanks for the awesome work you guys are doing to this life changing mod!

https://youtu.be/z3mwv-DbbpU

**Comment by Tensuko** [comment]
Picture with F5 active pls, so we can see the traffic boxes.
Also, what sprayer is this ?

**Comment by resultwrap** [comment]
They are all the built in game sprayers. No moded sprayers. 
1st sprayer - Horsh Leed PT 350
2nd sprayer - Stara Imperador 3.0
3rd sprayer - Hardi Rubicon 9000
4th sprayer - Hardi Navigator 6000


![20210116140124_1](https://user-images.githubusercontent.com/59973904/104820674-863a1900-5804-11eb-8a70-b80a58e1a572.jpg)
![20210116140323_1](https://user-images.githubusercontent.com/59973904/104820685-93570800-5804-11eb-9627-7b700c1ecd92.jpg)
![20210116140542_1](https://user-images.githubusercontent.com/59973904/104820688-95b96200-5804-11eb-9683-9a2c6130b0c2.jpg)
![20210116140721_1](https://user-images.githubusercontent.com/59973904/104820690-98b45280-5804-11eb-96fd-69711c6eef53.jpg)
![20210116140217_1](https://user-images.githubusercontent.com/59973904/104820693-9c47d980-5804-11eb-8726-d9975f22b49c.jpg)

Does this all look correct?

**Comment by Tensuko** [comment]
No. F5 only once. that screen we need. yellow boxes.

**Comment by resultwrap** [comment]
I don't know what you mean...

**Comment by resultwrap** [comment]
Okay, I figured it out, I will get those pictures.

**Comment by resultwrap** [comment]
![20210116165805_1](https://user-images.githubusercontent.com/59973904/104824086-f3f23f00-581c-11eb-84b7-04bb55c1744d.jpg)
![20210116165952_1](https://user-images.githubusercontent.com/59973904/104824088-f5bc0280-581c-11eb-92be-543c152632db.jpg)
![20210116170117_1](https://user-images.githubusercontent.com/59973904/104824089-f785c600-581c-11eb-9394-ae8cb431d2b3.jpg)
![20210116170234_1](https://user-images.githubusercontent.com/59973904/104824090-f94f8980-581c-11eb-9ca4-86d88ff5c451.jpg)

**Comment by Tensuko** [comment]
Hmm yea thats Issue of the Vehicles, just like Rubicon.

@pvaiko seems like we have to generaly do something here ? Or also something for the planed XML file ?

**Comment by Tensuko** [comment]
That should be tweaked a while ago:
* Introduced ignoreCollisionBoxesWhenFolded

To ignore too wide collision boxes for configured vehicles when
unfolded.
